components/ShippingRates/InfoModal.vue

```diff
@@ -0,0 +1,192 @@
+          {{ modalTitle }}
+          <UButton
+            color="gray"
+            variant="ghost"
+            icon="i-heroicons-x-mark-20-solid"
+            @click="closeModal"
+          />
+        </div>
+      </template>
+
+      <component
+        :is="hasMultipleShippingRates ? 'ul' : 'div'"
+        v-if="shippingRates.length"
+        class="space-y-4"
+      >
+        <component
+          :is="hasMultipleShippingRates ? 'li' : 'div'"
+          v-for="(s, index) in shippingRates"
+          :key="s.index"
+        >
+          <UCard
+            :ui="{
+              body: { base: 'space-y-4' },
+              footer: { base: 'flex justify-end items-center' },
+            }"
+          >
+            <UFormGroup
+              label="Name of this shipping option"
+              :ui="{ container: 'space-y-2' }"
+            >
+              <UInput
+                :value="s.nameEn"
+                placeholder="Shipping option name"
+                :disabled="isViewMode"
+                @input="(e) => handleInputShippingRates(e, 'nameEn', index)"
+              />
+              <UInput
+                placeholder="運送選項名稱"
+                :value="s.nameZh"
+                :disabled="isViewMode"
+                @input="(e) => handleInputShippingRates(e, 'nameZh', index)"
+              />
+            </UFormGroup>
+
+            <UFormGroup label="Price(USD) of this shipping option">
+              <UInput
+                :value="s.price"
+                type="number"
+                step="0.01"
+                :min="0"
+                :disabled="isViewMode"
+                @input="(e) => handleInputShippingRates(e, 'price', index)"
+              />
+            </UFormGroup>
+
+            <template v-if="hasMultipleShippingRates && isEditMode" #footer>
+              <UButton
+                label="Delete"
+                variant="outline"
+                color="red"
+                @click="deleteShippingRate(index)"
+              />
+            </template>
+          </UCard>
+        </component>
+        <div class="flex justify-center">
+          <UButton
+            v-if="isEditMode"
+            label="Add Options"
+            variant="outline"
+            icon="i-heroicons-plus-20-solid"
+            @click="addMoreShippingRate"
+          />
+        </div>
+      </component>
+      <div v-else class="flex justify-center items-center w-full gap-[8px] py-[36px]">
+        <span>
+          No items
+        </span>
+        <UButton
+          v-if="isEditMode"
+          label="Add Options"
+          variant="outline"
+          icon="i-heroicons-plus-20-solid"
+          @click="addMoreShippingRate"
+        />
+      </div>
+
+      <template #footer>
+        <div class="flex justify-end items-center">
+          <UButton v-if="isEditMode" label="Save" @click="handleOnSave" />
+          <div v-if="isViewMode" class="flex justify-end items-center gap-[8px]">
```

Comment:
Should this be v-else-if?

Reply:
[💄 Display checkbox section if is not editMode](https://github.com/likecoin/nft-book-press/pull/88/commits/688511d89e42940c3aeda8c1d0454b5bf490d008)